FreshRSS/FreshRSS
# fixed: Mapco/Ansum theme

Closes #6735

Changes proposed in this pull request:

- fixed: line height of left handside feed navigation
- fixed: css map comment
- improved (similar to https://github.com/FreshRSS/FreshRSS/pull/6446): hovering area over the drop down category icon is now bigger

How to test the feature manually:

1. see the feed navigation in normal view
2. hover over the icons and categories
3. see the icon while hovering

Pull request checklist:
